gradle.properties

```diff
@@ -1,10 +1,16 @@
 kotlin.code.style=official
-kotlin.mpp.enableGranularSourceSetsMetadata=true
-kotlin.native.enableDependencyPropagation=false
 kotlin.js.generate.executable.default=false
-
+kotlin.native.binary.memoryModel=experimental
+kotlin.incremental.js=true
+kotlin.incremental.multiplatform=true
+kotlin.native.ignoreIncorrectDependencies=true
+kotlin.native.ignoreDisabledTargets=true
+kotlin.mpp.stability.nowarn=true
+kotlin.mpp.enableCompatibilityMetadataVariant=true
+kotlin.mpp.enableCInteropCommonization=true
+kotlin.internal.mpp.hierarchicalStructureByDefault=true
 
 # versions
 coroutines_version=1.6.0
 atomicfu_version=0.17.0
-kotlin_version=1.6.10
+kotlin_version=1.7.0
```

Comment:
minor: I prefer version catalogs

Reply:
will migrate later, will be hard to rebase now